great_expectations_cloud/agent/config.py

```diff
@@ -1,9 +1,9 @@
-from great_expectations.compatibility.pydantic import AnyUrl, BaseSettings
-from great_expectations.data_context.cloud_constants import CLOUD_DEFAULT_BASE_URL
+from great_expectations.compatibility.pydantic import BaseSettings
 
 
 class GxAgentEnvVars(BaseSettings):  # type: ignore[misc] # BaseSettings is has Any type
     # pydantic will coerce this string to AnyUrl type
-    gx_cloud_base_url: AnyUrl = CLOUD_DEFAULT_BASE_URL
+    # gx_cloud_base_url: AnyUrl = CLOUD_DEFAULT_BASE_URL
+    gx_cloud_base_url: str
```

Comment:
Why are you changing this?

Reply:
I changed that since we don't allow the use of Any types anymore with one of the linting changes that was made.

Reply:
That was a change in a FE code not here.
Regardless the `Any` type is something we want to avoid where possible, despite the name `AnyUrl` is not an `Any` type, and it is a much stronger type than `str`.

Please revert this.